### PR TITLE
DO -919

### PR DESCRIPTION
**Pull request by Hedgehog18**
Display UK Company register on Data Sets page.

### CODE REVIEW

File: src/components/pages/datasets/datasets.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ const datasets = {
   '/api/fop': { urlEnding: 'fop/' },
   '/api/pep/': { urlEnding: 'pep/' },
   '/api/pep': { urlEnding: 'pep/' },
+  '/api/company/uk/': { urlEnding: 'uk/' },
```
Comment (roman-tiukh) [inline src/components/pages/datasets/datasets.js:12]:
```suggestion
  '/api/company/uk/': { urlEnding: 'uk-companies/' },
```

File: src/components/pages/datasets/company/UKcompanyList.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import Table from 'components/table/Table';
+import PageBox from 'components/pages/PageBox';
+import { useTranslation } from 'react-i18next';
+
+
+const UKcompanyList = () => {
```
Comment (roman-tiukh) [inline src/components/pages/datasets/company/UKcompanyList.jsx:8]:
```suggestion
const UKCompanyList = () => {
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by Hedgehog18 — COMMENTED** [review]
Changed the name to "UKCompanyList", wherever it is used. Fixed "UrlEnding".

File: src/components/pages/datasets/DatasetsRoutes.jsx [hunk 3]
```diff
@@ -9,12 +9,18 @@ import CompanyList from './company/CompanyList';
...
+import UKCompanyList from './company/UKCompanyList';
 
 const DatasetsRoutes = (props) => {
   const { match } = props;
 
   return (
     <Switch>
+      <Route
+        exact
+        path={`${match.path}:id/uk/`}
```
Comment (roman-tiukh) [inline src/components/pages/datasets/DatasetsRoutes.jsx:21]: ........
```suggestion
        path={`${match.path}:id/uk-companies/`}
```

File: src/components/pages/datasets/company/UKCompanyList.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    { header: t('name'), prop: 'name' },
+    { header: t('status'), prop: 'status' },
+    { header: t('address'), prop: 'address' },
+    { header: t('registrationDate'), prop: 'registration_date' },
+  ];
+  return (
+    <PageBox header={t('viewTheRegistry')} noBox>
+      <Table
+        columns={columns}
+        url="uk/"
```
Comment (roman-tiukh) [inline src/components/pages/datasets/company/UKCompanyList.jsx:21]: is this working url?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
Are you testing your code before push commits in repo?

File: src/components/pages/datasets/company/UKCompanyList.jsx (hunk 4 above)
Comment (Hedgehog18) [reply src/components/pages/datasets/company/UKCompanyList.jsx:21]: yes, it works

Comment (Hedgehog18) [reply src/components/pages/datasets/company/UKCompanyList.jsx:21]: Everything works for me without errors, I rechecked it several times.

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
the backend does not have this endpoint, we cannot be sure that everything works until we test it on real data

**Pull request closed without merging**